File: confreader/starter-code/c#/ConfReader.cs
using System;
using System.Collections.Generic;

public class ConfReader {

	public IDictionary<String, Object> Parse() {
		string file = System.IO.File.ReadAllText(@"ENTER FILE PATH HERE");

		// Implementation here
		IDictionary<String, Object> parsedMap = new Dictionary<String, Object>();  
		return parsedMap;
	}
}

File: confreader/starter-code/javascript/ConfReader.js
class ConfReader {
	
	function parse(file, callback) {
		//Implementation here
		var parsedMap = {};
  		if (!file) {
    		return;
  		}
  		var reader = new FileReader();
  		reader.addEventListener('load', function(e) {
    		var text = e.target.result;

    		callback(parsedMap);
		});
  		reader.readAsText(file);

  		
	}
}